src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -44,9 +45,6 @@
         + "and supply channel key '%s'.";
     private static final String CHANNEL_KEY_MAPPING_DOESNT_EXIST = "Failed to find supply channel of key '%s'.";
 
```

Comment:
in general handling exceptions in this class is very complex, split among many methods
some methods producing completion stages do not have .exceptionally() handlers, some of they have
i'm quite confused about the logic behind it